src/formatHit.js

```diff
@@ -31,7 +37,10 @@ export default function formatHit({
     const value = templates.inputValue(suggestion);
     const dropdownValue = templates.dropdownSuggestion({
       ...suggestion,
-      name: hit._highlightResult.locale_names[0].value
+      name: highlightedName,
+      administrative: highlightedAdministrative,
+      city: highlightedCity,
+      country: hit._highlightResult.country.value
```

Comment:
Ok, not sure to get it, we get the country (non highlight) with just hit.country no?